test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/CustomTestHostTests.cs

```diff
@@ -164,17 +163,18 @@ public void RunAllTestsWithMixedTFMsWillProvideAdditionalInformationToTheDebugge
         runEventHandler.Errors.Should().BeEmpty();
         testHostLauncher.AttachDebuggerInfos.Should().HaveCount(2);
         var targetFrameworks = testHostLauncher.AttachDebuggerInfos.Select(i => i.TargetFramework).ToList();
-        targetFrameworks.Should().OnlyContain(tfm => tfm == Framework.FromString("net451") || tfm == Framework.FromString("netcoreapp2.1"));
+        targetFrameworks.Should().OnlyContain(tfm => tfm.StartsWith(".NETFramework") || tfm.StartsWith(".NET "));
```

Comment:
Isn't it `.NETCoreApp`?

Reply:
Nope, the runtime name is nowadays return .NET <version>. They don't recommend parsing it to get version, so I am checking on .NETFramework or .NET here. Even though .NET would be enough, I wanted to show that both are possible.